sources/HUBViewControllerImplementation.m

```diff
@@ -48,6 +48,8 @@
 #import "HUBActionHandler.h"
 #import "HUBViewModelDiff.h"
 
+CFTimeInterval const kDownloadingTimeTreshold = 0.07;
```

Comment:
Also make it `static`. And since it's only relevant for images add that information to the variable name. Like this:
`HUBImageDownloadTimeThreshold`.